tests/ci/ci.py

```diff
@@ -1284,7 +1284,8 @@ def _configure_docker_jobs(docker_digest_or_latest: bool) -> Dict:
     # FIXME: we need login as docker manifest inspect goes directly to one of the *.docker.com hosts instead of "registry-mirrors" : ["http://dockerhub-proxy.dockerhub-proxy-zone:5000"]
     #   find if it's possible to use the setting of /etc/docker/daemon.json (https://github.com/docker/cli/issues/4484#issuecomment-1688095463)
     docker_images_helper.docker_login()
-    missing_multi_dict = check_missing_images_on_dockerhub(imagename_digest_dict)
+    # missing_multi_dict = check_missing_images_on_dockerhub(imagename_digest_dict) #REMOVEME: force clickhouse to build integration
+    missing_multi_dict = {'altinityinfra/integration-test' : imagename_digest_dict['altinityinfra/integration-test'], 'altinityinfra/integration-tests-runner':imagename_digest_dict['altinityinfra/integration-tests-runner'], 'altinityinfra/integration-helper':imagename_digest_dict['altinityinfra/integration-helper']}
```

Comment:
Why? do we need to always rebuild those images for whatever reason?

Reply:
Not always, just once. Will be removed before final merge. 

Those images were amd only.